exercises/handling-errors/practice/src/activities.ts

```diff
@@ -0,0 +1,77 @@
+// TODO Part A: Import `ApplicationFailure` from `@temporalio/common`
+import { Address, Bill, Distance, OrderConfirmation } from './shared';
+import { log } from '@temporalio/activity';
+
+export async function getDistance(address: Address): Promise<Distance> {
+  log.info('getDistance invoked; determining distance to customer address');
+
+  // this is a simulation, which calculates a fake (but consistent)
```

Comment:
Nit: I don't like comments in informal lower case. I consider them part of the documentation. I'd do something more like:

// Calculate a fake-but-consistent distance for the customer address using character-count.
// The address must be invoked consistently without variations (St, Street, St., etc)